lib/browser/script.js

```diff
@@ -38,14 +38,12 @@ window.addEventListener('load', function() {
     return new Promise(function(resolve) {
       var current = window.domTabs[0];
       cleanUp(current);
+
       current.classList.add('current');
-      if (window.inTabsView) {
-        current.classList.add('in-tabs-view');
-      }
+      current.active = !window.inTabsView;
       current.style.zIndex = 1;
-
       current.style.top = 0;
-      current.querySelector('.frame').style.height = viewportHeight + 'px';
+      // current.querySelector('.frame').style.height = viewportHeight + 'px';
```

Comment:
we want the current tab to be sticky and start scrolling off screen once `previewHeight` of it's content is visible.
this is why we need to set the height on the container, to control when the current tab stops sticking.